docs/questions/query-builder/expressions/cumulativecount.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Cumulative count
+---
+
+# Cumulative count
+
+Cumulative count computes the additive total count of rows across a breakout.
+
+## Syntax
+
+You can use the pre-defined "Cumulative count" metric, or a custom expression:
+
+```
+CumulativeCount
+```
+
+Cumulative count can only be used in the query builder's **Summarize** step — you cannot use cumulative count to create a custom column or in a custom filter.
+
+## How cumulative count is computed
+
+Under the hood, cumulative count is computed in three steps:
+
+1. Group records using the breakout from the **Group by** block,
+2. Count the rows in each group,
+3. For each group, compute the cumulative total of the counts in all previous groups up to the current group.
+
+Because cumulative count uses values from previous rows, the sort order in the breakout column matters.
+
+For example, if you have a table with data grouped by month, cumulative count will be computed like this:
+
+| Month    | `Count` | `CumulativeCount` |
+| -------- | ------- | ----------------- |
+| July     | 5       | 5                 |
+| November | 4       | 5+4 = 9           |
```

Comment:
Maybe "May", "June", "July": shorter names, and sequential.

Reply:
I picked months intentionally so that either order makes sense, depending on where in the year you start: so july 2023, November 2023, March 2024 makes sense as the order, and March 2023, july 2023, Novemeber 2023 makes sense as the order.